src/items/alien-archives/copaxi_special_ops.json

```diff
@@ -776,7 +776,7 @@
p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.16Rhhb1Ta1reEMVD]{Blazing Orbit (Su)}</p>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.BehQeoPVENIv8Xg3]{Corona (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.J2gfUBr49IJr09qX]{Crush (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.y7y1SnuR9mbAj3Ux]{Crushing Force (Su)}</p>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.O0tTwXlYJ71u8XMY]{Curving Flare (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.xlDhH4ABvqHyAj44]{Defy Gravity (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.u0yi4o8dMLCDMSU1]{Draining Agony (Su)}</p>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.QpPsiuPYHIBL3V7p]{Glow of Life (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.72PoGdbd5Wx4E0B3]{Gravity Pulse (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.BUnYTBQAlPgETpAd]{Gravity Surge (Su)}</p>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.NrblHz9EK0wxJxED]{Hypnotic Glow (Su)}</p>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.AJHBDxoH0i3GuKN7]{Luminescent Inquisition (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.MrUxuOjsr9O1zMVj]{Reflection (Su)}</p>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.YaY7MBxIQAAaLU5X]{Vital Reinforcement (Su)}</p>\n<h2>10th Level</h2>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.sqVOS211ptbOIume]{Forceful Shield (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.6veb6wNRS3hYb76M]{Gravity Well (Su)}</p>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.nLGfiBPzBWuakbHM]{Solar Inferno (Su)}</p>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.5J7zDyEyZak5dTSs]{Soul Furnace (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.6NFx4YOYQxn7oD4J]{Stalwart Shield (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.X0x3JwGyEEJ3Ii6N]{Stealth Warp (Su)}</p>\n<h2>14th Level</h2>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.pjXxq9BSqDmVGUoQ]{Binary Shield System (Su)}</p>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.Ga72o407X6UZlmaP]{Gallant Rebuttal (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.4936gbpW7qjLvQKf]{Gravity Shield (Su)}</p>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.4EWgS85b5oEUJBxv]{Sunbolt (Su)}</p>\n<h2>16th Level</h2>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.dmuliDXdehEHv1Mv]{Superlative Constellation (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.OLXmy2N2DRCNDbDU]{Superlative Nebula (Su)}</p>\n<p>@Icon[type:graviton] @UUID[Compendium.sfrpg.class-features.MwQV4bUQ4OAWv4Yr]{Ultimate Graviton (Su)}</p>\n<p>@Icon[type:photon] @UUID[Compendium.sfrpg.class-features.SDInkeo4eWICc5cm]{Ultimate Photon (Su)}</p>"
```

Comment:
I have no idea what they're doing there, but can you remove the Chinese characters and replace them with `’`